community/plugins/spring-ai-alibaba-plugin-larksuite-starter/src/main/java/com/alibaba/cloud/ai/Configure/PluginToolAutoConfiguration.java

```diff
@@ -0,0 +1,21 @@
+package com.alibaba.cloud.ai.Configure;
+
+import com.alibaba.cloud.ai.properties.LarkSuiteProperties;
+import com.alibaba.cloud.ai.service.LarkSuiteService;
+import jdk.jfr.Description;
+
+
+/**
+ * @author 北极星
+ */
+@EnableConfigurationProperties({LarkSuiteProperties.class})
+public class PluginToolAutoConfiguration {
+
+    @Bean
+    @ConditionalOnMissingBean
+    @Description("LarkSuite OApi")
```

Comment:
Please use `org.springframework.context.annotation.Description` instead.

Reply:
The function description should be as detailed as possible, so LLM can better understand what purpose this function is designed for and to decide when to call this function.